slides/week5/week5.org

```diff
@@ -0,0 +1,85 @@
+#+TITLE: Week 5
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org, rortiz9@gatech.edu
+
+* What are we doing today?
+- Classes in C++
+- Handling merge conflicts in =git=
+
+* What are classes?
+- Definitions for objects, i.e. what data and methods they have
+ - In other words, a blueprint for how to create an object
+- An object is called an *instance* of a class
+ - An object has all the functions and data structures defined in its class
+ - It doesn't share any data with other objects
```

Comment:
I feel like this is misleading. Objects can share data with other objects. Perhaps something like "Objects of the same class share the same schema of data but the actual values stored are independent instance to instance"